index.bs

```diff
@@ -1878,13 +1877,32 @@ A {{WebTransportError}} has the following internal slots.
 
 <div algorithm>
 
+To <dfn for="WebTransportError">create</dfn> a {{WebTransportError}} error with a
+{{WebTransportErrorSource}} |source| and optionally a |streamErrorCode| defaulting
+to `null`, run these steps:
+
+1. Let |message| be an [=implementation-defined=] string.
+1. Let |options| be `{source, streamErrorCode}`.
+1. Return a new {{WebTransportError/constructor()}} with |message| and |options|.
```

Comment:
@domenic Is this method of creating the object okay? I thought we had to create the object and run the constructor steps separately, but maybe my knowledge is out-of-date.

Reply:
Indeed, no. You should not have your own "create" algorithm; you should use https://whatpr.org/webidl/1211.html#dfn-create-exception . The pattern is shown in https://whatpr.org/webidl/1211.html#example-domexception-derived-throwing (see also the sentence immediately before it)

Reply:
Thanks, so much nicer!